build.gradle.kts

```diff
@@ -29,6 +30,7 @@ dependencies {
     implementation("io.confluent", "kafka-protobuf-provider")
     // Our custom avro version. See https://github.com/ImFlog/avro
     implementation("com.github.ImFlog.avro", "avro", avroVersion)
+    implementation("com.google.code.gson","gson",jsonVersion)
```

Comment:
`gson` is already available as a transitive dependency from kafka-schema-registry-parent.
Thus there is no need to define the version.

Also i term of formatting, add spaces after the `,`.